package-rfcx/rfcx/_util.py

```diff
@@ -0,0 +1,10 @@
+import datetime
+
+
+def date_before(days=30):
+    return (datetime.datetime.utcnow() - datetime.timedelta(days=days)
+            ).replace(microsecond=0).isoformat() + 'Z'
+
+
+def date_now():
+    return datetime.datetime.utcnow().replace(microsecond=0).isoformat() + 'Z'
```

Comment:
I was going to ask "Why do you add `Z`?", but after googling I found that its because Python is ugly!